CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 All notable changes to this project will be documented in this file. This change
 log follows the conventions of [keepachangelog.com](http://keepachangelog.com/).
 
+## [0.7.0]
+### Added
+- Transaction history for an account
+- Transactions page to show transactions between two generations
+- Started using [middleware](https://github.com/aeternity/aepp-middleware)
```

Comment:
Rebased blockchain explorer on [aepp-middleware](https://github.com/aeternity/aepp-middleware)

Reply:
The meaning of re-based sounds unclear to me.